src/ansys/dynamicreporting/core/utils/enhanced_images.py

```diff
@@ -0,0 +1,482 @@
+"""
```

Comment:
For this version, I am only creating a group of functions. But in the long term, as more requirements are added, I can see the benefits of writing a class. For instance, output image dimensions are required by the users.